pkg/api/v1alpha1/cloudstackdatacenterconfig_types.go

```diff
@@ -28,12 +28,10 @@ type CloudStackDatacenterConfigSpec struct {
 
 	// Domain contains a grouping of accounts. Domains usually contain multiple accounts that have some logical relationship to each other and a set of delegated administrators with some authority over the domain and its subdomains
 	Domain string `json:"domain"`
-	// Zone is an organizational construct typically used to represent a single datacenter, and all its physical and virtual resources exist inside that zone. It can either be specified as a UUID or name
-	Zone CloudStackResourceRef `json:"zone"`
+	// Zones is a list of one or more zones that are managed by a single CloudStack management endpoint.
+	Zones []Zone `json:"zones"`
```

Comment:
If we are changing this to multiple zones, should we introduce a new type called `CloudStackMultiDatacenterConfig` or `CloudStackMultiZoneDatacenterConfig`?

Reply:
How do you see that being advantageous vs this approach? Thats a little loaded because I can't see the value and would vote KISS.

Reply:
I guess what I am confused by is that initially we went with the naming `CloudStackDatacenterConfig` because a zone is kind of considered a single datacenter when compared to vsphere. So I thought the plan was that when we supported multiple zones, it would be a different type of deployment. But I guess we are going to treat one management endpoint as a datacenter instead of one zone as a datacenter? @maxdrib

Reply:
yes, one cloudstack management endpoint is a datacenter.